deploy/vagrant/core/config_openstack.rb

```diff
@@ -3,14 +3,17 @@
 
 # OpenStack specific configurations go here
 
-def config_os(config, i, total, name)        
+def config_os(config, i, total, name, version)
   config.vm.box = "dummy"
   config.vm.box_url = 
     "https://github.com/cloudbau/vagrant-openstack-plugin/raw/master/dummy.box"
   # Make sure the private key from the key pair is provided
   config.ssh.private_key_path = KEY_PATH
-  config.vm.synced_folder "../../", "/tachyon", type: "rsync", 
-         rsync__exclude: ["../../.git/", "shared/"]
+  
+  if version == "Local"
+    config.vm.synced_folder "../../", "/tachyon", type: "rsync", 
+           rsync__exclude: ["../../.git", "shared/"]
+  end
```

Comment:
Oh so if the version is local there will be two synced folders (/tachyon and /vagrant) in the vm. There will be some overlap (since /tachyon will contain /vagrant) but its necessary because of the src=/vagrant/files later?